### PR TITLE
Updating the README installation and setup instructions.

### PR DESCRIPTION
**Pull request by weerd**
This pull request does some cleanup to the README and updates the project installation and setup instructions!

It also adds instructions for Python installation via Homebrew ✨

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -42,64 +42,211 @@ Script Name | Functionality
...
+
+### Step 4: Install DBT dependencies
+
+Once the DBT profile is setup, head into the `dbt` directory and install all DBT [package dependencies](https://docs.getdbt.com/docs/building-a-dbt-project/package-management):
+
+```shell
+$ ~/Code/quasar/quasar/dbt
+```
+
+> Note: The above is not a typo, there is a `quasar` directory inside the main repository `quasar` directory. We know. Don't ask... 🤷‍♂️
```
Comment (katiecrane) [inline README.md:205]: This note is KEY haha - this was my downfall during my setup!

File: README.md [hunk 2]
```diff
@@ -42,64 +42,211 @@ Script Name | Functionality
...
+```shell
+$ ~/Code/quasar/quasar/dbt
 
-## Troubleshooting
-[TROUBLESHOOTING.md](/docs/troubleshooting.md)
+$ pipenv run dbt serve
+```
 
-## Deployment
-[DEPLOYMENT.md](/docs/deployment.md)
```
Comment (katiecrane) [inline README.md:99]: Are these easy enough to find that we don't need to link to them?

**Review by katiecrane — APPROVED** [review]
WOOHOO this is a work of art!! So detailed and so helpful!

File: README.md (hunk 1 above)
Comment (weerd) [reply README.md:205]: 😏

File: README.md (hunk 2 above)
Comment (weerd) [reply README.md:99]: Oh! Nope, good call. I think During my edit I totally overwrote the troubleshooting, deployment and credit sections! Good catch @katiecrane! I'll restore them 😅

**Review by DFurnes — APPROVED** [review]
This is fantastic! Nice work Diego!! 📚

**Pull request merged**